src/test/resources/adjustment.sample

```diff
@@ -2,11 +2,10 @@
   "credit": 1.0,
   "discount": 1.0,
   "insurance": 0.5,
-  "invoice_id": "2d6d495c-c237-476f-aa48-57f7cbf74ca4",
+  "invoiceId": "2d6d495c-c237-476f-aa48-57f7cbf74ca4",
```

Comment:
All the .sample fields for the sub objects can be deleted

Reply:
All unnecessary .sample test data deleted. Remaining files renamed to use '-' instead of '_'.